sensor_msgs/Image face
string call
string message
uint8 obj_id